denizen_scripts/survival/repo_link/systems/world_barriers.dsc

```diff
@@ -0,0 +1,8 @@
+World_barriers:
+  type: world
+  debug: false
+  events:
+    on player enters outermost_barrier:
+      - teleport <player> <player.location.sub[<context.area.center>].normalize.mul[10].find.surface_blocks.within[3].first>
```

Comment:
`<player>` is default here when the queue is attached to the player; we can safely remove this argument
```suggestion
      - teleport <player.location.sub[<context.area.center>].normalize.mul[10].find.surface_blocks.within[3].first>
```